cves/CVE-2014-7809.yml

```diff
@@ -32,7 +32,7 @@ announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE data.
   Please enter your date in YYYY-MM-DD format.
-announced:
+announced: 2014-12-10
```

Comment:
Apache seems to have an announcement that predates this date in accordance with their wiki at https://cwiki.apache.org/confluence/display/WW/S2-023 

- *Date:* 2014-11-21 by Lukasz Lenart

Reply:
Changes addressed.